README.md

```diff
@@ -24,6 +24,15 @@ For an example of how to install/import see:
 import_install_example.sh
 ```
 
+## Common settings
+
+Variables/Options which can be set/exported to change the behaviour of the
+library and various functions during execution.
+
+interactive - true/false - en/dis-ables interactive promps and features requiring a TTY/PTY
+strict_mode - true/false - en/dis-ables hell options to stop and/or exit on errors and undefined variables
```

Comment:
`hell` options? lol

Reply:
I think I rather like hell options:

strict_mode
interactive

Both seem like options I'd like to define when in hell.